src/Package/Impl/StatusBar/VsStatusBar.cs

```diff
@@ -21,49 +21,32 @@ internal class VsStatusBar : IStatusBar {
         private readonly IApplicationShell _shell;
         private ItemsControl _itemsControl;
         private Visual _visualRoot;
-        private EnvDTE.DTEEvents _dteEvents;
 
         [ImportingConstructor]
         public VsStatusBar(IApplicationShell shell) {
             _shell = shell;
         }
 
-        private Visual GetRootVisual() {
+        private void UpdateRootVisual() {
             var dte = _shell.GetGlobalService<EnvDTE.DTE>();
```

Comment:
I believe this is no longer needed, IVsUIShell.DialogOwnerWindow works just fine (it does return top level HWND)